test/test_scanner.py

```diff
@@ -666,6 +666,12 @@ def test_binaries(self, binary, package, version, are_in, not_in):
                         "gnutls-cli",
                         "2.3.11",
                     ),
+                    (
+                        "https://ftp.openbsd.org/pub/OpenBSD/OpenSSH/",
+                        "openssh-7.9.tar.gz",
```

Comment:
This looks like source code when I open it.  The cve-**bin**-tool won't detect source code (it only works on binary files and won't even scan text files right now), so this test is likely invalid.

Reply:
I have updated this test. Please let me know if the changes which I made are correct.